File: OmkarFCUBEAPI/FreightMasters/Models/DistanceMasterTripModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FreightMasters.Models
{
    public class DistanceMasterTripModel
    {
        public string? MasterID { get; set; }
        public string? ValidFrom { get; set; }
        public string? ValidUpto { get; set; }
        public string? FromLocation { get; set; }
        public string? LoggedInUser { get; set; }
        public List<DistanceDetailTripModel> DistanceDetailsTripList { get; set; }
    }
}

File: OmkarFCUBEAPP/src/app/models/distancemastertripmodel.ts
export class Distancemastertripmodel {
    masterID: string = "";
    validFrom: string = "";
    validUpto: string = "";
    fromLocation: string = "";
    locationName: string = "";
    loggedInUser: string = "";
    distanceDetailsTripList: Distancedetailstripmodel[] = [];
    
}

export class Distancedetailstripmodel {
    distanceDtlID: string = "";
    masterID: string = "";
    fromLocation: string = "";
    toLocation: string = "";
    toLocationName: string = "";
    fromLocationName: string = "";
    kms: string = "";
    index: string = "";
    enrouteExpTruck: string = "";
    enrouteExpTrailer: string = "";
    enrouteExpCarCarrier: string = "";
    enrouteExpEmpty: string = "";
    enrouteExpRemarks: string = "";
    definedTollExp: string = "";
    
}
